### PR TITLE
Fix uninstall using openshift_prometheus_state=absent

### PR DESCRIPTION
**Pull request by zgalor**
This was broken in https://github.com/openshift/openshift-ansible/pull/6811

bz: https://bugzilla.redhat.com/show_bug.cgi?id=1540806

### CODE REVIEW

**Comment by zgalor** [comment]
@mjudeikis PTAL

File: playbooks/openshift-prometheus/private/uninstall.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@
   - name: Run the Prometheus Uninstall Role Tasks
     include_role:
       name: openshift_prometheus
-      tasks_from: uninstall
+      tasks_from: uninstall_prometheus
```
Comment (sdodson) [inline playbooks/openshift-prometheus/private/uninstall.yml]: I think the problem is that it's not specifying the full filename `uninstall.yaml` or now `uninstall_prometheus.yaml` after you've renamed it.

Comment (sdodson) [reply playbooks/openshift-prometheus/private/uninstall.yml]: example https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml#L312-L314

Comment (mjudeikis) [reply playbooks/openshift-prometheus/private/uninstall.yml]: this means we have this all over the place:
https://github.com/openshift/openshift-ansible/blob/afe031ec2f3db2f014c79ee16c5dc93d1b3e3199/playbooks/openshift-management/private/uninstall.yml

Comment (mjudeikis) [reply playbooks/openshift-prometheus/private/uninstall.yml]: Plus: we might want to enforce pattern: uninstall.yaml in entry point playbooks, and uninstall_productname in roles? or smth? as now its all different too

Comment (zgalor) [reply playbooks/openshift-prometheus/private/uninstall.yml]: no. the problem was https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/main.yaml#L28 calling uninstall_prometheus which was renamed.
Now I tested both uninstalling with the uninstall.yml playbook or with config.yml and `openshift_prometheus_state=absent` and both work

Comment (mjudeikis) [reply playbooks/openshift-prometheus/private/uninstall.yml]: yes, I missed that one entry point... my bad...

Comment (zgalor) [reply playbooks/openshift-prometheus/private/uninstall.yml]: @mjudeikis if you think this is better, I can keep the playbooks named uninstall and change the name here: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/main.yaml#L28

Comment (mjudeikis) [reply playbooks/openshift-prometheus/private/uninstall.yml]: I dont have opinion about this. 
Just with 3.9 we might get some strict standard about naming. anything @sdodson will say :) and we say "this is how we do things" we can tide up things later :)

**Pull request merged**